File: Member/LiuHaixin/qingguo-request/QingGuo/interfaceTestcase/scene1.py
import unittest

from Utils.Sendhttp import SendHttp


class scene1(unittest.TestCase):
    def setUp(self):
        self.url = "/common/skuList"
    def test_skulist_success1(self):
        result = SendHttp().sent_get(self.url)
        print(result)
    def test_skulist_success2(self):
        Id = {"goodsId": 1}
        result = SendHttp().sent_get(self.url, Id)
        print(result)
    def test_skulist_success3(self):
        Id = {"goodsId": 2}
        result = SendHttp().sent_get(self.url, Id)
        print(result)
    def test_skulist_success4(self):
        Id = {"goodsId": 3}
        result = SendHttp().sent_get(self.url, Id)
        print(result)
if __name__=='__main__':
    unittest.main()

File: Qing_Guo/src/com/webtest/htmlunit_demo/Scene1.java
package com.webtest.htmlunit_demo;
import org.testng.annotations.Test;

import com.webtest.core.HtmlUnit;
import com.webtest.utils.ReadPro;
/*×÷Õß£ºÎ¤Óî½Ü*/
public class Scene1 {
	@Test
	public void skulist() throws Exception {
		String result = HtmlUnit.doGet(ReadPro.getProValue("url")+"/common/skuList");
		System.out.println(result);
	}
	@Test
	public void skulist_goodsId1() throws Exception {
		String result = HtmlUnit.doGet(ReadPro.getProValue("url")+"/common/skuList","goodsId=1");
		System.out.println(result);
	}
	@Test
	public void skulist_goodsId2() throws Exception {
		String result = HtmlUnit.doGet(ReadPro.getProValue("url")+"/common/skuList","goodsId=2");
		System.out.println(result);
	}
	@Test
	public void skulist_goodsId3() throws Exception {
		String result = HtmlUnit.doGet(ReadPro.getProValue("url")+"/common/skuList","goodsId=3");
		System.out.println(result);
	}
}
